packages/main/src/Table.ts

```diff
@@ -504,7 +504,7 @@ class Table extends UI5Element {
 	}
 
 	_isFeature(feature: any) {
-		return Boolean(feature.onTableActivate && feature.onTableAfterRendering);
+		return Boolean(feature.onTableActivate || feature.onTableAfterRendering);
```

Comment:
Maybe it makes sense to move all of this _isFeature, _getSelection stuff into the util. I'll have a look for that